test-support/src/main/java/org/springframework/ldap/test/LdapTestUtils.java

```diff
@@ -109,15 +109,6 @@ public static void startEmbeddedServer(int port, String defaultPartitionSuffix,
     }
 
     /**
-     * @deprecated use {@link #startEmbeddedServer(int, String, String)} instead.
```

Comment:
I'm not sure I'm comfortable removing this method in a minor release. Is this necessary?